src/capi/azext_capi/custom.py

```diff
@@ -161,7 +161,9 @@ def create_aks_management_cluster(cmd, cluster_name, resource_group_name=None, l
                              "✓ Created AKS management cluster", "Couldn't create AKS management cluster")
     os.environ[MANAGEMENT_RG_NAME] = resource_group_name
     with Spinner(cmd, "Obtaining AKS credentials", "✓ Obtained AKS credentials"):
-        command = ["az", "aks", "get-credentials", "-g", resource_group_name, "--name", cluster_name]
+        logger.warning("aks credentials will overwrite existing cluster config for cluster %s if exist", cluster_name)
```

Comment:
```suggestion
        logger.warning("AKS credentials will overwrite the cluster config for cluster %s if it already exists", cluster_name)
```
Does logging this warning here play nice with the Spinner's updating message? I was wondering if this should be moved before the Spinner line...I guess I can try it out.

Reply:
good call. I'll make that update to move it outside the spinner

Reply:
moved it outside the spinner